cmd/plugin.go

```diff
@@ -166,15 +166,49 @@ Example:
 	return cmd
 }
 
-type skipReason struct {
-	plugin string
-	reason string
+type installReport struct {
+	Skipped        bool
+	Plugin         string
+	SkipReason     string
+	DocURL         string
+	Version        string
+	IsUpdateReport bool
 }
 
-func (u *skipReason) String() string {
-	ref := ociinstaller.NewSteampipeImageRef(u.plugin)
+func (ir *installReport) SkipString() string {
+	ref := ociinstaller.NewSteampipeImageRef(ir.Plugin)
 	_, name, stream := ref.GetOrgNameAndStream()
-	return fmt.Sprintf("Plugin:   %s\nReason:   %s", fmt.Sprintf("%s@%s", name, stream), u.reason)
+
+	return fmt.Sprintf("Plugin:   %s\nReason:   %s", fmt.Sprintf("%s@%s", name, stream), ir.SkipReason)
+}
+
+func (ir *installReport) InstallString(isUpdate bool) string {
```

Comment:
can we move installreport into its own file installreport.go, maybe under `display`?